core/state/statedb.go

```diff
@@ -131,6 +131,10 @@ func (self *StateDB) Reset(root common.Hash) error {
 	return nil
 }
 
+func (self *StateDB) GetTrie() Trie {
```

Comment:
I added this because I couldn't see any existing way to get access to the trie from the statedb

Reply:
You can do `statedb.Database().OpenTrie()`